.gitignore

```diff
@@ -184,3 +184,6 @@ cmake-build-*
 
 #Python
 __pycache__
+Marlin/Untitled.code-workspace
+Marlin/Configuration.h
+Marlin/Configuration.h
```

Comment:
`Configuration.h` shouldn’t be in the `.gitignore` because this file is also updated as Marlin is updated. Any changes done in a fork would also be ignored.